modules/trimgalore/main.nf

```diff
@@ -52,13 +54,15 @@ process TRIMGALORE {
             $c_r1 \\
             $tpc_r1 \\
             ${prefix}.fastq.gz
+
         cat <<-END_VERSIONS > versions.yml
         "${task.process}":
             trimgalore: \$(echo \$(trim_galore --version 2>&1) | sed 's/^.*version //; s/Last.*\$//')
             cutadapt: \$(cutadapt --version)
         END_VERSIONS
         """
     } else {
+        outprefix = "*val*.fq.gz"
```

Comment:
Does `val` point to something, or does trimgalore output it as `val`? (again just curious to learn more about trim galor)

Reply:
Sure does! 

```
--basename <PREFERRED_NAME>	Use PREFERRED_NAME as the basename for output files, instead of deriving the filenames from
                        the input files. Single-end data would be called PREFERRED_NAME_trimmed.fq(.gz), or
                        PREFERRED_NAME_val_1.fq(.gz) and PREFERRED_NAME_val_2.fq(.gz) for paired-end data. --basename
                        only works when 1 file (single-end) or 2 files (paired-end) are specified, but not for longer lists.
```

Reply:
TrimGalore automatically names paired-end output files using `*val_1.fq.gz` and `val_2.fq.gz`.